core/src/main/java/io/grpc/internal/CensusStatsModule.java

```diff
@@ -356,7 +356,7 @@ public void outboundMessage(int seqNo) {
       this.parentCtx = checkNotNull(parentCtx);
       TagValue methodTag = TagValue.create(fullMethodName);
       this.startCtx = module.tagger.toBuilder(parentCtx)
-          .put(DeprecatedCensusConstants.RPC_METHOD, methodTag)
+          .putPropagating(DeprecatedCensusConstants.RPC_METHOD, methodTag)
```

Comment:
For backwards-compatibility these tags are still treated as propagating tags. We could make them non-propagating (they should be non-propagating eventually) though that may be breaking. WDYT?

Reply:
Although I don't know anyone who depends on this being propagated (gRPC doesn't), it's safer to keep the current behavior in this PR, and change it to non-propagating in a separate PR.

Reply:
SGTM, I'll send a follow-up PR for it.